DBMS/DBMS3/assignment1.sql

```diff
@@ -0,0 +1,16 @@
+-- to display book information of those books issued more than two months
+SELECT m.member_name,m.member_id,t.title_name,bi.accession_no,bi.issue_date,bi.due_date
```

Comment:
ACTION: 
1. Requirement is to include no of months in retrive columns, Please do that
2. Requirement is to sort result on member_nm and member_title, please do that